### PR TITLE
[KEYCLOAK-19847] - Restore Docker Dist tests coverage

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

**Review by pedroigor — COMMENTED** [review]
@andreaTP Is it really fixing the cause or a workaround to make tests pass?

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java [hunk 1]
```diff
@@ -51,6 +51,5 @@ void failIfDevProfile(LaunchResult result) {
                 () -> "The Error Output:\n" + result.getErrorOutput() + "doesn't contains the expected string.");
         assertTrue(result.getErrorOutput().contains("For more details run the same command passing the '--verbose' option. Also you can use '--help' to see the details about the usage of the particular command."),
                 () -> "The Error Output:\n" + result.getErrorOutput() + "doesn't contains the expected string.");
-        assertEquals(4, result.getErrorStream().size());
```
Comment (pedroigor) [inline quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java:54]: This is actually making sure we don't introduce regressions. Change tests like that usually is not a good idea for this reason.

Or do you have something better in mind?

Our requirement is to make sure we have specific messages are shown only once.

Comment (andreaTP) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java:54]: I'll try to have a more generic test for this 👍

**Comment by andreaTP** [comment]
@pedroigor no workarounds, but the log messages are collected differently using the raw distribution and Docker. I'll try to find a compromise that will perform the check without depending on this nuance.

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java (hunk 1 above)
Comment (andreaTP) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java:54]: @pedroigor should be ok now 🙂

**Comment by pedroigor** [comment]
@andreaTP There we go! Thanks.

**Review by pedroigor — APPROVED** [review]

**Pull request merged**